### PR TITLE
Register miskode.is-a.dev

### PR DESCRIPTION
**Pull request by miskode**
Register miskode.is-a.dev CNAME from miskode.github.io

### CODE REVIEW

File: domains/miskode.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "miskode",
+        "email": "miskonx@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "miskode.github.io"
```
Comment (CuteDog5695) [inline domains/miskode.json:8]: 404 page

**Review by iostpa — DISMISSED** [review]

File: domains/miskode.json (hunk 1 above)
Comment (miskode) [reply domains/miskode.json:8]: It is no longer a 404, content has been uploaded.

**Review by triach-rold — APPROVED** [review]
Website is complete and reachable ! Had fun playing :)

**Review by MaskDuck — DISMISSED** [review]

File: domains/miskode.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "miskode",
+        "email": "miskonx@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "miskode.github.io",
+        "TXT": "e03000495c20c26b2f53651111d57d"
```
[the remark this answers is not in the document]
Comment (miskode) [reply domains/miskode.json]: Deleted txt

**Comment by miskode** [comment]
Is this ever going to be merged?

**Comment by DEV-DIBSTER** [comment]
It's been a single day, you need to be patient to say the least.

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**